Gruntfile.js

```diff
@@ -190,29 +222,6 @@ module.exports = function(grunt) {
         },
 
         /*
-         * commits the ia.js version file and package.json
-         * still needs to be pushed
-         */
-        gitcommit: {
-            ia_pages: {
-                options: {
-                    message: 'Release IA pages version: <%= pkg.version %>'
-                },
-                files: {
-                    src: [
-                        static_dir + 'js/ia0.*.0.js',
-                        static_dir + 'js/ddgc0.*.0.js',
-                        static_dir + 'css/ddgc0.*.0.css',
-                        static_dir + 'css/ia0.*.0.css',
-                        static_dir + 'css/ddgc.css',
-                        'package.json'
```

Comment:
I'm not using gitcommit anymore, I'm using an exec task instead: https://github.com/duckduckgo/community-platform/pull/1083/files#diff-35b4a816e0441e6a375cd925af50752cR270
As you can see package.json is included there